index.js

```diff
@@ -81,12 +77,37 @@ function getid(arr) {
 function createNewTask() {
     let id = getid(tasksList);
     id++;
-    const task  = {
-        id: "" + id, 
-        text: "", 
-        completed: false 
-};
+    const task = {
+        id: "" + id,
+        text: "",
+        completed: false
+    };
     tasksList.push(task);
-    console.log(tasksList);
 };
-createNewTask()
\ No newline at end of file
+
+function deleteTask() {
+    const ul = document.querySelector('.todo-list');
+    ul.onclick = function (event) {
+        if (event.target.className != "destroy") return;
+        let li = event.target.closest('li');
+        let liId = li.id;
+        if (!li) {
+            return;
+        } else {
+            let task = tasksList.find(item => item.id == liId);
+            for (let i = 0; i < tasksList.length; i++) {
```

Comment:
`for` is ok, but there is another way of deleting an item from array. 

tasksList = tasksList.filter(t => t.id !== task.id);